### PR TITLE
wxwidgets: Update to v3.2.6 

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Release notes can be found [here](https://raw.githubusercontent.com/wxWidgets/wxWidgets/v3.2.6/docs/readme.txt).

**Test Plan**
- Rebuilt veracrypt against this.
- Resolves https://github.com/getsolus/packages/issues/4442

**Checklist**

- [X] Package was built and tested against unstable
- [x] This change could gainfully be listed in the weekly sync notes once merged (Write an appropriate message in the Summary section)

### CODE REVIEW

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
While you're here, could you add `monitoring.yml` files to both packages?

**Review by EbonJaeger — DISMISSED** [review]
LGTM, if someone wants to merge it. I don't know that I'll get to it today.

**Comment by davidjharder** [comment]
Sadly it seems rebuilds might be a good idea, I can't get `codelite` to build with new wxwidgets

**Comment by davidjharder** [comment]
You have a conflict with `cemu` because it was updated in the time since this PR was started. You will have a similar issue with `audacity`. You will need to rebase

File: packages/f/freefilesync/package.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ setup      : |
 build      : |
     %make -C Source exeName=FreeFileSync
     %make -C Source/RealTimeSync exeName=RealTimeSync
-install    : |
+install    : |2
```
Comment (malfisya) [inline packages/f/freefilesync/package.yml:39]: What happens here?

Comment (androidnisse) [reply packages/f/freefilesync/package.yml:39]: No idea, just did a bump, good catch.

**Review by malfisya — COMMENTED** [review]
Nitpick: Please remove "Summary" from the commit message when no message needed

**Comment by malfisya** [comment]
I grep the repo and found 3 packages is still missing from the rebuild:

- 0ad
- trelby
- prusa-slicer

File: packages/r/radiotray-ng/abi_libs [hunk 2]
```diff
@@ -1,2 +1 @@
-radiotray-ng
```
Comment (malfisya) [inline packages/r/radiotray-ng/abi_libs:1]: This is kinda alarming. Can you test `radiotray-ng` and make sure it behaves normally?

File: packages/r/radiotray-ng/abi_symbols [hunk 3]
```diff
@@ -1,7 +1 @@
-radiotray-ng:__bss_start
-radiotray-ng:_edata
-radiotray-ng:_end
-rtng-bookmark-editor:__bss_start
-rtng-bookmark-editor:_edata
-rtng-bookmark-editor:_end
```
Comment (malfisya) [inline packages/r/radiotray-ng/abi_symbols:6]: Alarming. See above.

File: packages/v/visualboyadvance-m/abi_symbols [hunk 4]
```diff
@@ -1,8 +1,2 @@
-vbam:BC
-vbam:PC
-vbam:SP
 vbam:main
-visualboyadvance-m:BC
-visualboyadvance-m:PC
-visualboyadvance-m:SP
```
Comment (malfisya) [inline packages/v/visualboyadvance-m/abi_symbols:7]: Alarming. See above.

**Review by malfisya — CHANGES_REQUESTED** [review]
There are ABI removal in `radiotray-ng` and `visualboyadvance-m`. Can you make sure both of them run normally?

Also as noted in my previous comment, there are still 3 packages missing from the rebuild:
- 0ad
- trelby
- prusa-slicer

File: packages/r/radiotray-ng/abi_libs (hunk 2 above)
Comment (androidnisse) [reply packages/r/radiotray-ng/abi_libs:1]: Works fine. Listened to smooth jazz.

File: packages/v/visualboyadvance-m/abi_symbols (hunk 4 above)
Comment (androidnisse) [reply packages/v/visualboyadvance-m/abi_symbols:7]: Started up as expected. No errors in terminal.

**Comment by androidnisse** [comment]
> There are ABI removal in `radiotray-ng` and `visualboyadvance-m`. Can you make sure both of them run normally?
> 
> Also as noted in my previous comment, there are still 3 packages missing from the rebuild:
> 
>     * 0ad
> 
>     * trelby
> 
>     * prusa-slicer

0ad and prusa-slicer does not build with current recipe. I have opened two issues for them.

**Comment by EbonJaeger** [comment]
Is this just waiting for 0ad?

**Comment by davidjharder** [comment]
I would say no. Oad doesn't build at the moment, but it's not a wxwidgets issue, as far ass i can tell

**Comment by androidnisse** [comment]
> Is this just waiting for 0ad?



> Not really. Oad doesn't build at the moment, but it's not a wxwidgets issue, as far as i can tell

So can this go in now? :)

**Review by malfisya — CHANGES_REQUESTED** [review]
I am inclined to get this in soon, but why wxPython get updated? Any reason?

**Review by malfisya — APPROVED** [review]
Verified wxPython update does not affect its revdeps.

**Pull request merged**